yahoo_fantasy_api/team.py

```diff
@@ -113,28 +113,29 @@ def _compact_eligible_pos(j):
             pass
         return roster
 
-    def change_positions(self, day, modified_lineup):
+    def change_positions(self, modified_lineup, day=None, week=None):
```

Comment:
We can't reorder the parameters because that would break the API. Instead, we have two options:
1. Introduce a new API that takes in the week, such as `change_positions_for_week`.
2. My preferred approach is to treat the day parameter as either a `datetime` object, preserving its previous behavior, or as a week. We can use `isinstance` to check if it's a `datetime` object and handle it accordingly.